src/main/java/seedu/address/logic/commands/TrainingCommand.java

```diff
@@ -28,15 +30,15 @@
 
     public static final String TRAINING_ADD_SUCCESS = "Training successfully added.";
 
-    private Date date;
+    private String date;
     private List<Index> indexList;
 
     public TrainingCommand(List<Index> indexList) {
-        this.date = new Date();
+        this.date = new Date().toString();
         this.indexList = indexList;
     }
 
-    public TrainingCommand(Date date, List<Index> indexList) {
+    public TrainingCommand(String date, List<Index> indexList) {
```

Comment:
Just curious - why are there 2 constructors for `TrainingCommand`?

When is which constructor used?